easybuild/tools/testing.py

```diff
@@ -138,7 +138,8 @@ def session_state():
     }
 
 
-def create_test_report(msg, ecs_with_res, init_session_state, pr_nrs=None, gist_log=False, easyblock_pr_nrs=None):
+def create_test_report(msg, ecs_with_res, init_session_state, pr_nrs=None, gist_log=False, easyblock_pr_nrs=None,
+                       ec_parse_error=None):
     """Create test report for easyconfigs PR, in Markdown format."""
```

Comment:
Bonus points if you extend the docstring to document the parameters ;)

Reply:
done in c93efc2